config/locales/devise/en.yml

```diff
@@ -50,13 +50,18 @@ en:
         it. Please follow the instructions below.
       start:
         accordion: You will also need
+        bullet_1_img: Email
         bullet_1_html: An <strong>email address</strong>.
+        bullet_2_img: Device
         bullet_2_html: Enable <strong>two-step authentication</strong>.
```

Comment:
I think it's fine here since we're following an established pattern, but if we were to do the original implementation over again, I think it'd be nicer and more maintainable to structure these bullets as an array (easier to add and remove).

e.g.

```yml
start: 
  bullets: 
    - image: Email
      html: An <strong>email address</strong>
    - image: Device
      html: Enable <strong>two-step authentication</strong>.
```